src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Channels.cs

```diff
@@ -9,13 +9,28 @@ public class ScenarioTests_Channels : MaestroScenarioTestBase
         private readonly string repoName = "maestro-test1";
         private readonly string branchName;
         private readonly string branchNameWithRefsHeads;
+        private TestParameters _parameters;
```

Comment:
Doesn't this class have the same "two tests running in parallel sharing parameters" problem?

Reply:
It's NUnit, and there are 2 tests in the class, so it does have this problem.

Reply:
My understanding is that the attributes handle that, unless I read this wrong:
https://github.com/nunit/docs/wiki/SetUp-and-TearDown-Changes

Reply:
Or does it potentially share the instance despite the attribute?

Reply:
If that's the case then I can move the instance into the test case, though I'm not sure what kind of guarantees we have on clean up in that case.

Reply:
nothing on that page suggests that it changes the "one instance of the test class for all tests" property on NUnit. This field is shared between all tests.

Reply:
Decided to split the test case into a new class so that teardown can handle the clean up (it's the only test category that has more than one test case anyway).